Assets/Scripts/Screeps3D/RoomObjects/Views/ExtractorView.cs

```diff
@@ -33,19 +32,11 @@ private void Update()
         {
             if (_extractor == null)
                 return;
-
-            if (null != anim)
+         
+            if (_extractor.Cooldown > 0)
             {
-                // play Bounce but start at a quarter of the way though
-                //anim.Play("Bounce", 0, 0.25f);
-                // Sadly I can't get the animation to work, only a Default Take seems to get imported, and when an animation controller is attached, the model dissappears
```

Comment:
Was the animation for the extractor removed? I wrote this back before we updated unity, the animation should work now I think?